webapp/store/views.py

```diff
@@ -172,7 +172,7 @@ def search_snap():
 
         if "total" in searched_results:
             total_results_count = searched_results["total"]
-            total_pages = ceil(total_results_count / size)
+            total_pages = ceil(total_results_count / 44)
```

Comment:
Search results and category results have a different number of results on the page (hence the `size` variable) - setting this as a static number breaks one or the other

Reply:
As far as I can see, both search results and category results display 44 snaps per page, except first page of category results. Or am I missing something?